RestaurantFunction.cs

```diff
@@ -1,132 +1,126 @@
+using System;
 using System.Net;
 using System.Net.Http;
+using System.Threading.Tasks;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.Http;
 using Microsoft.Azure.WebJobs.Host;
-using MongoDB.Driver;
 using MongoDB.Bson;
-using System.Threading.Tasks;
 using MongoDB.Bson.Serialization;
+using MongoDB.Driver;
+
 
 namespace MongoDB.Tutorials.AzureFunctions
 {
     public static class RestaurantFunction
     {
         [FunctionName("Restaurant")]
-        public static async Task<HttpResponseMessage> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", "patch", "delete", Route = "Restaurant/id/{restaurantId}")]HttpRequestMessage req, string restaurantId, TraceWriter log)
+        public static Task<HttpResponseMessage> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", "patch", "delete", Route = "Restaurant/id/{restaurantId}")]HttpRequestMessage req, string restaurantId, TraceWriter log)
         {
             log.Info("Restaurant function processed a request.");
-            string returnValue = string.Empty;
-            HttpStatusCode returnStatusCode = HttpStatusCode.Forbidden;
 
-            try
-            {
-                string strMongoDBAtlasUri = System.Environment.GetEnvironmentVariable("MongoDBAtlasURI");
-                log.Info($"Atlas connection string is: {strMongoDBAtlasUri}");
-
-                MongoUrl mongoUrl = new MongoUrl(strMongoDBAtlasUri);
-                var settings = MongoClientSettings.FromUrl(mongoUrl);
-                //for more on why we're using ServerSelectionTimeout, read https://scalegrid.io/blog/understanding-mongodb-client-timeout-options/
-                settings.ServerSelectionTimeout = new System.TimeSpan(0, 0, 5);
+            var strMongoDBAtlasUri = Environment.GetEnvironmentVariable("MongoDBAtlasURI");
```

Comment:
This is the same code from the other function. Any reason not to put this in a common place and use it in both places?